tasks/models.py

```diff
@@ -9,20 +9,14 @@
 class Session(SettingModel):
     order = models.PositiveIntegerField()
 
-    # Fields with respect to Tasks
-    tasks_number = models.PositiveIntegerField(
-        _("Hoeveel taken worden er binnen deze sessie bij de deelnemer afgenomen?"),
-        null=True,
+    repeats = models.PositiveBigIntegerField(
+        _("Hoe vaak wordt deze sessie herhaald?"),
```

Comment:
Suggestion:
"Hoe vaak zal de participant deze sessie doorgaan?"

But then using proper Dutch/English

In addition, a help-text with more details could be worth looking at.